pages/en/network/developing.mdx

```diff
@@ -2,36 +2,39 @@
 title: Developing
 ---
 
-Developers are a core part of The Graph. They are responsible for both building and querying subgraphs, unlocking data for the web3 ecosystem.
+Developers are a core part of The Graph ecosystem. They are responsible for both building and querying subgraphs, unlocking data for the web3 ecosystem.
 
 ## Building Subgraphs
 
-Developers that build subgraphs use the [GraphQL Schema Language](https://graphql.org/learn/schema-language/) to define the data they want to make available.
+Developers can build subgraphs, open APIs, and deploy them locally or in the [Subgraph Studio](https://thegraph.com/studio/).
 
-The Graph CLI tool streamlines subgraph creation. When you initialize your subgraph, all the default files will be created & configured. Developers can use the built-in APIs or add other libraries as dependencies as long as they are compatible with [AssemblyScript](https://github.com/AssemblyScript/assemblyscript). Subgraphs should be thoroughly tested before deployment. Developers can use [MatchStick](https://github.com/LimeChain/matchstick) to test the mapping logic in a sandboxed environment. Once your subgraph is built & tested, it can be deployed to The Graph's decentralized network. However, before anyone can begin querying it, your subgraph must be curated & indexed.
+Once published and indexed to the The Graph Network, anyone can query the subgraph with [GraphQL](https://graphql.org/learn/schema-language/) and use it in their protocol or applications.
 
 ### Curation
 
-Curators use their knowledge of web3 to assess and signal new subgraphs. They look for subgraphs with reliable and relevant data.
+Subgraph developers often curate their own subgraph. This proves to Indexers that they've published a high-quality subgraph to the network and attracts additional Curators.
```

Comment:
I think the key thing here is that Subgraph developers will curate so that indexers will pick up the subgraph (rather than necessarily being about high quality). As you say, it may attract additional curators

Reply:
Fixed that - thank you!